### ISSUE TITLE
Checklist and coordination for v3.7.2 patch release

### ISSUE THREAD

**Issue by jywarren** [publiclab/image-sequencer#2037]
Also see:

- https://github.com/publiclab/image-sequencer/issues/2038

* [x] open an issue using the "release" template with this checklist with title `Checklist and coordination for v0.0.0 major/minor/patch release` (see [semantic versioning](https://docs.npmjs.com/about-semantic-versioning/))
* [ ] create a release [project](https://github.com/publiclab/image-sequencer/projects) from [this template](https://github.com/publiclab/image-sequencer/projects/5). You can copy a project from its menu.
* [x] compile release notes below from corresponding [release project](https://github.com/publiclab/image-sequencer/projects).
* [x] open a pull request with updated version numbers
    * [x] update version number in `package.json` (ex #1695)
    * [x] update version number in `examples/sw.js` (ex #1734)
    * [x] run `npm install` to update `package-lock.json` (from recent node version - 16 at time of writing, in GitPod should work)
    * [x] check in `package-lock.json`
* [x] finalize and merge to `main` branch (freeze merges to `main` branch until next step)

Now, move to `stable` branch:

* [x] force push from `main` to `stable`
* [x] then in `stable` branch, compile `/dist/` files with `grunt build`
* [x] add `/dist/` files with `git add -f /dist/*` and commit them to `stable` branch
* [x] publish `stable` branch to `npm` with `npm publish` (logging in first as necessary)

Draft a release:

* [x] [create a release on GitHub](https://github.com/publiclab/image-sequencer/releases) and use features description + release notes from below
* [x] tag version number branch (i.e. `v0.0.0`) based on `stable` or choose `stable`
* [x] publish to live Github pages [demo](https://sequencer.publiclab.org) (with [bash script](https://github.com/publiclab/image-sequencer/pull/1703) from `/scripts/update-demo`) (from within GitPod works well)
* [x] move anything necessary to next release project, i.e. https://github.com/publiclab/image-sequencer/projects/[insert project number]
* [x] close this issue!

Noting we're now in this process in https://github.com/publiclab/image-sequencer/issues/1751 for `v3.7.0`.

****

### Release notes
Compile and edit release notes below, to be copied into the release description.

#### Fixed

- https://github.com/publiclab/image-sequencer/pull/1589
- https://github.com/publiclab/image-sequencer/pull/2041 (high priority) - fixes node-png dep issue

#### Changed

-

**Comment by jywarren** [comment]
![Screen Shot 2022-01-15 at 2 25 29 PM](https://user-images.githubusercontent.com/24359/149635381-568bfa56-7db1-4504-bd8b-6cd101dba9ce.png)
![Screen Shot 2022-01-15 at 2 25 42 PM](https://user-images.githubusercontent.com/24359/149635386-d8ae09e6-9980-4074-b291-60713a87ffe9.png)

**Comment by harshkhandeparkar** [comment]
Hi! I saw the activity. I didn't get why we published 3.7.1 without publishing 3.7.0 though.

For the next release, I guess we can start with updating https://github.com/publiclab/image-sequencer/projects/4.

**Comment by jywarren** [comment]
We just merged https://github.com/publiclab/image-sequencer/pull/2041 and anticipate it will unblock uptake of IS in MapKnitter: Bump image-sequencer from 1.4.1 to 3.7.1 https://github.com/publiclab/mapknitter/pull/1585

So, i'm inclined to release a `v3.7.2` with just this addition, but we could also wait for more from here. If we wait, it will probably be `v3.8.0`, given some new features.

**Comment by jywarren** [comment]
I'm going to go ahead and release `v3.7.2` with these:

- https://github.com/publiclab/image-sequencer/pull/1589
- https://github.com/publiclab/image-sequencer/pull/2041 (high priority) - fixes node-png dep issue

I'll move the rest into a new issue for `v3.8.0`!

**Comment by jywarren** [comment]
Released! https://github.com/publiclab/image-sequencer/releases/tag/v3.7.2

Published! https://www.npmjs.com/package/image-sequencer

Demo updated too!

<img width="275" alt="image" src="https://user-images.githubusercontent.com/24359/159585456-f8b4fe5a-5224-44a0-99f3-eb566eb60f99.png">

**Comment by jywarren** [comment]
Unfortunately the dist files didn't make it in!

**Comment by jywarren** [comment]
I'm not sure how. Compiling and releasing as 3.7.3, unfortunately 😭

**Comment by jywarren** [comment]
Published as v3.7.3!
